packages/manager/modules/adp/src/deploy/review.html

```diff
@@ -0,0 +1,138 @@
+                <div class="form-group row">
+                    <label for="public_cloud"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_cloud_project_name">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="public_cloud" data-ng-bind="$ctrl.selectedPublicCloud.description"></span>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="shh_key"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_cloud_project_shh_key">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="shh_key">
+                            {{ ::$ctrl.selectedSshKey.name ? $ctrl.selectedSshKey.name : $ctrl.selectedSshKey.id }}
+                        </span>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="private_network"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_cloud_project_private_network">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="private_network">
+                            {{ ::$ctrl.vRack.data.name ? $ctrl.vRack.data.name : $ctrl.vRack.data.id }}
+                        </span>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="region"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_cloud_project_region">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="region" data-ng-bind="$ctrl.adp.os_region"></span>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label for="admin_mail"
+                        class="col-sm-3 col-form-label col-form-label-sm"
+                        data-translate="adp_deploy_review_cloud_project_email">
+                    </label>
+                    <div class="col-sm-9 mb-1">
+                        <span class="form-control-plaintext border-0" id="admin_mail" data-ng-bind="$ctrl.adp.admin_mail"></span>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </section>
+    <!-- nodes config -->
+    <section class="mb-3">
+        <h6 data-translate="adp_deploy_review_nodes_title"></h6>
+        <div class="border-top">
```

Comment:
Could you refer to my previous message?

Reply:
The entire form has been replaced by `<dl>` element